src/dotnet/SnippetGenerator/SnippetGenerator.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
For new things comming into the tools repo it would be good if you put them under the "tools/" folder as we want to clean-up the "src/" folder overtime